aws/common/sqs.tf

```
@@ -26,8 +26,18 @@ resource "aws_sqs_queue" "notify_internal_tasks_queue" {
   name = "${var.celery_queue_prefix}notify-internal-tasks"
   # This queue was created outside of terraform and has this value set to false in staging and production.
   sqs_managed_sse_enabled = false
-  # This queue was created outside of terraform and has this value set to 310 in staging and production.
-  visibility_timeout_seconds = 310
+  # This queue was created outside of terraform and has this value set to default in staging and production.
+  visibility_timeout_seconds = ${var.sqs_visibility_timeout_default}
```

Curly brackets on non interpolated variables is deprecated in TF :)

```suggestion
  visibility_timeout_seconds = var.sqs_visibility_timeout_default
```